### ISSUE TITLE
Persistence implementation incomplete

### ISSUE THREAD

**Issue by koalatux** [sysgrok/rs-matter-stack#2]
It looks like the persistance implementation is at least in the `light_eth` example not complete. Storing seems to work when [this line](https://github.com/ivmarkov/rs-matter-stack/blob/ba19dfd45a7626956c33539d8c60de0354be8332/src/lib.rs#L388) is changed, but loading is not implemented. Persistence works in the `onoff_light` example of `rs-matter`.

**Comment by ivmarkov** [comment]
Yes, unfortunately. I'm neck deep into implementing memory optimizations and some commissioning-related improvements to the underlying `rs-matter`. I hope in a month or so to be able to get back to this.

In the meantime, you could try to fix it and open a PR.

**Comment by ivmarkov** [comment]
Persistence does work since  commit https://github.com/sysgrok/rs-matter-stack/commit/ec0bff50eb8c8cf0cf71384f293bce50c3511f6e

Also, on device stop and restart, the controller successfully re-establishes connection and re-subscribes to the already-provisioned device.
